perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json

```diff
@@ -29,14 +29,14 @@
         },
         "cache-dir": "/tmp/cache"
       },
-      "branch": "read_cache_release",
-      "end_date": "2024-03-31 05:30:00+00:00"
+      "branch": "master",
+      "end_date": "9999-12-31 05:30:00+00:00"
     },
     {
-      "config_name": "master_benchmark_with_gRPC_changes_enabled",
+      "config_name": "master_benchmark_with_gRPC_changes_enabled_20240408",
```

Comment:
@raj-prince Please take a look as this test is failing, you might require to change anything from config.

Reply:
No, I wanted to disable this test until the fix of deadline_timeout issue.
I'll disable this from the KOKORO config side only.